### PR TITLE
Update release notes Influxdb 1.5.2 OSS and Enterprise

### PR DESCRIPTION
**Pull request by stevebang**
Update release notes and Enterprise release links.

### CODE REVIEW

File: content/influxdb/v1.5/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ menu:
...
+* Check for root user when running `buildtsi`.
+* [1.5] Adjustable TSI Compaction Threshold.
+
+### Bugfixes
+
+* backport: check for failure case where backup directory has no manifest files.
+* Fix regression to allow `now()` to be used as the group by offset again.
+* 1.5: Revert `Use MADV_WILLNEED when loading TSM files`.
+* Ignore index size in Engine.DiskSize().
+* [1.5] Fix `buildtsi` partition key.
```
Comment (sanderson) [inline content/influxdb/v1.5/about_the_project/releasenotes-changelog.md]: Any reason for the difference in formatting between `[1.5]` in this line and `1.5:` in line 21?

File: content/influxdb/v1.5/about_the_project/whats_new.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ menu:
 
 > For a comprehensive list of the changes between versions, see [InfluxDB's Changelog](/influxdb/v1.5/about_the_project/releasenotes-changelog/).
 
-Included here are some highlights of the InfluxDB 1.5 release:
+Included here are some highlights of the InfluxDB 1.5.x release:
```
Comment (sanderson) [inline content/influxdb/v1.5/about_the_project/whats_new.md]: With a wildcard patch number here, I think "release" should be "releases".

File: content/influxdb/v1.5/administration/upgrading.md [hunk 3]
```diff
@@ -9,23 +9,23 @@ menu:
 
 This page includes the steps required for upgrading InfluxDB OSS and InfluxDB Enterprise from earlier versions.
 
-* [Upgrading from 1.3/1.4 (no TSI Preview) to 1.5 (TSI enabled)](#upgrading-from-13-14-no-tsi-preview-to-15-tsi-enabled)
-* [Upgrading from 1.4 (TSI Preview enabled) to 1.5 (TSI enabled)](#upgrading-from-13-14-tsi-preview-enabled-to-15-tsi-enabled)
-* [Upgrading from 1.3 to 1.5 (TSI enabled)](#upgrading-from-13-to-15-tsi-enabled)
+* [Upgrading from 1.3/1.4 (no TSI Preview) to 1.5.x (TSI enabled)](#upgrading-from-13-14-no-tsi-preview-to-15-tsi-enabled)
```
Comment (sanderson) [inline content/influxdb/v1.5/administration/upgrading.md]: Update the link anchor to include the `x` added in the headings below.

`upgrading-from-13-14-no-tsi-preview-to-15x-tsi-enabled`

File: content/influxdb/v1.5/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (stevebang) [reply content/influxdb/v1.5/about_the_project/releasenotes-changelog.md]: That's the way it was in the 1.5.2 product release notes, and I missed it.

File: content/influxdb/v1.5/about_the_project/whats_new.md (hunk 2 above)
Comment (stevebang) [reply content/influxdb/v1.5/about_the_project/whats_new.md]: Good catch -- hurried too much.

**Pull request merged**